### ISSUE TITLE
Feature request, Docker WSS/TLS with Let's Encrypt Support

### ISSUE THREAD

**Issue by JCLengeling** [fonoster/routr#142]
Dear all,

thank you very much for the project.  It's working great within my local network running on my local docker server. My next step is to run it globally and expose the server to the internet. Obviously I want to do this securely and do this running the appropriate certificates. However, the docker setup doesn't seem to be suitable for this. Thus following feature request: 

Either:
1. Support for one of the well know docker containers that take care of automatically generating Letsencrypt certificates.  On my private server I am actually already doing this for my personal website relying on jwilder/nginx-proxy and nginxproxy/acme-companion. However, I assume that this isn't as easy due to working with Java Keystore for which the mentioned docker containers don't seem to be setup.
Or
2. Modification of the existing Dockerfile. If some appropriate environment variables are set in the docker compose/as part of the docker command  (domain name, email, certificate pw, ...), then install certbot jdk, .... and other dependencies, and afterwards essentially run the instructions specified in the guide on the website (https://routr.io/docs/guides/securing-the-signaling-path/).

Thank you very much for consideration, 
Kind regards

**Comment by psanders** [comment]
I know not having support for Let's Encrypt is a pain. 

Unfortunately, Routr v1 will not get any more updates beyond maintenance.

My recommendation is to create a sidecar to convert your certificates.

Here is an example of how to convert the certificate to work with Routr https://gist.github.com/psanders/0f89d59ce0bacadfb017c651e1c2f3de

I'm closing this issue since this will be supported in [v2](https://github.com/fonoster/routr/tree/v2).

**Comment by JCLengeling** [comment]
@psanders thank you very much for the reply and recommendation. I will give it a try. 

Nevertheless, is there a roadmap/timeline for the release of v2? Is the release just around the corner or did the v2 project just start?

**Comment by psanders** [comment]
@JCLengeling 

The first beta release is scheduled for the end of September. Would you like to beta test v2?

**Comment by JCLengeling** [comment]
@psanders yeah sure, that sounds like fun.  Please let me know what you like me to do :) (you may also reach me by mail if it's more extensive [joel.cedric@lengeling.eu](mailto:joel.cedric@lengeling.eu)).  

During the last week of September I am out of office, but during the start of October I am available to test.

**Comment by JCLengeling** [comment]
@psanders I am back in the office; if you need help, please let me know.
